draft-ietf-ice-rfc5245bis.xml

```diff
@@ -3461,557 +3441,761 @@ into the Completed state, and agent R can also send data.
-each client to the STUN server. In a basic voice-only IPv4 VoIP
-deployment, there will be four transactions per call (one for RTP and
-one for RTCP, for both caller and callee). Each transaction
-is a single request and a single response, the former being 20 bytes
-long, and the latter, 28. Consequently, if a system has N users, and
-each makes four calls in a busy hour, this would require N*1.7bps. For
-one million users, this is 1.7 Mbps, a very small number (relatively
-speaking).
+Another point of brittleness in classic STUN and any other unilateral
+mechanism is its absolute reliance on an additional server. ICE makes
+use of a server for allocating unilateral addresses, but allows
+agents to directly connect if possible. Therefore, in some cases, the
+failure of a STUN server would still allow for a call to
+progress when ICE is used.
 </t>
 
 <t>
-TURN traffic is more substantial. The TURN server will see traffic
-volume equal to the STUN volume (indeed, if TURN servers are deployed,
-there is no need for a separate STUN server), in addition to the
-traffic for the actual data. The amount of calls requiring
-TURN for data relay is highly dependent on network topologies, and
-can and will vary over time. In a network with 100% behave-compliant
-NAT, it is exactly zero. At time of writing, large-scale consumer
-deployments were seeing between 5 and 10 percent of calls requiring
-TURN servers. Considering a voice-only deployment using G.711 (so 80
-kbps in each direction), with .2 erlangs during the busy hour, this is
-N*3.2 kbps. For a population of one million users, this is 3.2 Gbps,
-assuming a 10% usage of TURN servers.
+Another point of brittleness in classic STUN is that it assumes
+that the STUN server is on the public Internet. Interestingly, with
+ICE, that is not necessary. There can be a multitude of STUN servers
+in a variety of address realms. ICE will discover the one that has
+provided a usable address.
+</t>
+
+<t>
+The most troubling point of brittleness in classic STUN is that it
+doesn't work in all network topologies. In cases where there is a
+shared NAT between each agent and the STUN server, traditional STUN
+may not work. With ICE, that restriction is removed.
+</t>
+
+<t>
+Classic STUN also introduces some security
+considerations. Fortunately, those security considerations are also
+mitigated by ICE.
+</t>
+
+<t>
+Consequently, ICE serves to repair the brittleness introduced in
+classic STUN, and does not introduce any additional brittleness
+into the system.
+</t>
+
+<t>
+The penalty of these improvements is that ICE increases session
+establishment times.
 </t>
 
 </section>
 
-<section title="Gathering and Connectivity Checks">
+<section title="Requirements for a Long-Term Solution">
+
+<t>From RFC 3424, any UNSAF proposal needs to provide:
+</t>
+
+<t><list style="hanging"><t>
+... requirements for longer term, sound technical solutions
+-- contribute to the process of finding the right longer term
+solution.
```

Comment:
What's this "..." and "--" stuff?

Reply:
Seems to be a copy/paste error. The text in RFC 3424 says:

       "Identify requirements for longer term, sound technical solutions;
       contribute to the process of finding the right longer term
       solution."